src/renderer/widgets/IWidget.hpp

```diff
@@ -1,8 +1,10 @@
 #pragma once
 
-#include "../../helpers/Vector2D.hpp"
+#include "../../helpers/Math.hpp"
 #include <string>
 
+#define VEC2CONVERT(vec) Vector2D(vec.x, vec.y)
+
```

Comment:
This is just temporary.
What would be the best way to convert Hyprlang::VEC2 to Vector2D in all the widgets?

Reply:
write a custom ctor out-of-line? is that allowed in C++? if not, that or a fn.

Reply:
you can't sadly. I added a fn.